08-rails/controllers.md

```diff
@@ -73,12 +80,12 @@ Rails.application.routes.draw do
 end
 ```
 
-Here we can see the last thing that `rails generate` built for us: a route! What this tells the router to do is:
-- When it receives a GET request...
+What we've told the router to do is:
+- When you receive a GET request...
 - ...to the `/books` path...
 - ...go to the `BooksController` class and run the `index` action
 
-We'll talk more about how the router works later in the week.
+That'll do for now. We'll talk more about how the router works later in the week.
 
 ## Fleshing Out the Controller
```

Comment:
The code below is using the outdated controller code that doesn't have your constant from the previous lesson